### PR TITLE
[FIX] remove utcfromtimestamp usage

### PR DESCRIPTION
**Pull request by Xavier-Do**
First commit: Doesn't look like an issue since it is a fallback in case `action.last_run` is not set
Second commit: to verify

### CODE REVIEW

**Comment by Xavier-Do** [comment]
@robodoo rebase-ff

**Review by Julien00859 — APPROVED** [review]
@robodoo r+

**Comment by Xavier-Do** [comment]
@robodoo skipci

**Review by Julien00859 — COMMENTED** [review]
@robodoo r-

**Review by Julien00859 — CHANGES_REQUESTED** [review]

File: odoo/addons/base/models/ir_binary.py [hunk 1]
```diff
@@ -223,7 +223,7 @@ def _get_image_stream_from(
             stream.etag += f'-{width}x{height}-crop={crop}-quality={quality}'
 
         if isinstance(stream.last_modified, (int, float)):
-            stream.last_modified = datetime.utcfromtimestamp(stream.last_modified)
+            stream.last_modified = datetime.datetime.fromtimestamp(stream.last_modified, tz=datetime.timezone.utc)
```
[the remark this answers is not in the document]
Comment (Julien00859) [reply odoo/addons/base/models/ir_binary.py:226]: + `from datetime import timezone`

**Comment by Julien00859** [comment]
@robodoo r+

**Comment by Xavier-Do** [comment]
@robodoo skipci

**Pull request closed without merging**